src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -168,21 +170,61 @@ private static SnapshotBuilderProgramDataListOption generateSnapshotBuilderProgr
                       Person.RACE_CONCEPT_ID,
                       "Race",
                       List.of(new SnapshotBuilderProgramDataListItem().id(43).name("unused 3")))))
-          .featureValueGroups(
-              List.of(
-                  new SnapshotBuilderFeatureValueGroup().name("Condition"),
-                  new SnapshotBuilderFeatureValueGroup().name("Observation"),
-                  new SnapshotBuilderFeatureValueGroup().name("Procedure"),
-                  new SnapshotBuilderFeatureValueGroup().name("Surveys"),
-                  new SnapshotBuilderFeatureValueGroup().name("Person")))
           .datasetConceptSets(
               List.of(
+                  new SnapshotBuilderDatasetConceptSet()
+                      .name("Drug")
+                      .table(
+                          new SnapshotBuilderTable()
+                              .datasetTableName(DrugExposure.TABLE_NAME)
+                              .primaryTableRelationship("fpk_person_drug")
+                              .secondaryTableRelationships(
+                                  List.of(
+                                      "fpk_drug_concept",
+                                      "fpk_drug_type_concept",
+                                      "fpk_drug_route_concept",
+                                      "fpk_drug_concept_s"))),
+                  new SnapshotBuilderDatasetConceptSet()
+                      .name("Condition")
+                      .table(
+                          new SnapshotBuilderTable()
+                              .datasetTableName(ConditionOccurrence.TABLE_NAME)
+                              .primaryTableRelationship("fpk_person_condition")
+                              .secondaryTableRelationships(
+                                  List.of(
+                                      "fpk_condition_concept",
+                                      "fpk_condition_type_concept",
+                                      "fpk_condition_status_concept",
+                                      "fpk_condition_concept_s"))),
+                  new SnapshotBuilderDatasetConceptSet()
+                      .name("Procedure")
+                      .table(
+                          new SnapshotBuilderTable()
+                              .datasetTableName(ProcedureOccurrence.TABLE_NAME)),
```

Comment:
Do we want to allow `SnapshotBuilderTable` objects to have no `primaryTableRelationships` or `secondaryTableRelationships`?

Reply:
Yeah, I didn't populate it here because it isn't used for tests, but based on your other comment, I think we could make primary table relationship required, so then I'd need to populate that here

Reply:
Is there a reason for other non-omop datasets that we wouldn't want to make this required?

Reply:
Discussed offline - its tricky to represent the ideal state so we're just leaving them optional for now.